contracts/drip/BalanceDrip.sol

```diff
@@ -11,7 +10,6 @@ library BalanceDrip {
 
   struct UserState {
```

Comment:
This should be refactored since it is now only a single variable

Reply:
I'd like to leave it like this just to leave it open for any additional fields.